src/app/lib/analyticsUtils/frontpage.js

```diff
@@ -0,0 +1,36 @@
+import deepGet from '../utilities/deepGet';
+
+export const getPageIdentifier = frontpageData =>
+  deepGet(['metadata', 'analyticsLabels', 'counterName'], frontpageData) ||
+  'unknown.page';
+
+const guidRegex =
+  '([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})';
+const curieRegex = new RegExp(
+  `http://www.bbc.co.uk/asset/${guidRegex}/desktop/domestic`,
+);
+export const getContentId = frontpageData => {
+  const curie = deepGet(['metadata', 'locators', 'curie'], frontpageData);
+  if (!curie) {
+    return null;
+  }
+  const matches = curie.match(curieRegex);
```

Comment:
Why do we need to do this? Is there a case where we expect a different format?

Reply:
Are you asking why I'm not splitting the `curie` on `/`s and picking the 4th? No major reason beyond input validation.

Reply:
Hmmm ok, it fails safe so lets see how we go, need to keep an eye on it with new services